qutebrowser/browser/commands.py

```diff
@@ -1444,7 +1444,8 @@ def download(self, url=None, dest_old=None, *, mhtml_=False, dest=None):
         else:
             qnam = tab.networkaccessmanager()
             download_manager.get(self._current_url(), user_agent=user_agent,
-                                 qnam=qnam, target=target)
+                                 qnam=qnam, target=target,
+                                 title=self._current_title())
```

Comment:
I think passing a title here makes `get_request` needlessly complicated for only this single use case.

Instead, you should be able to construct a `downloads.FileDownloadTarget` here (similar to what's done above when `--dest` is given), and pass that to `.get()` instead. Then there shouldn't be any changes in `qtnetworkdownloads.py` needed at all.